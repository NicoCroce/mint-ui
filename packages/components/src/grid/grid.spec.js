import React from "react";
import { render } from "@testing-library/react";
import Row from "./row";
import Col from "./col";

const Demo = () => (
	<Row>
		<Col sm={4}>Some value</Col>
		<Col sm={4}>Another value</Col>
		<Col sm={4}>Third value</Col>
	</Row>
);
test("Row Col renders correctly", () => {
	const { asFragment } = render(<Demo />);
	expect(asFragment()).toMatchSnapshot();
});
