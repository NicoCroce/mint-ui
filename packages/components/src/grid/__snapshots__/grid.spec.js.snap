// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row Col renders correctly 1`] = `
<DocumentFragment>
  <div
    class="row__Row-sc-1j7iuup-0 kHyiAL"
  >
    <div
      class="col__Col-sc-6sbgfs-0 hjGsrv"
    >
      Some value
    </div>
    <div
      class="col__Col-sc-6sbgfs-0 hjGsrv"
    >
      Another value
    </div>
    <div
      class="col__Col-sc-6sbgfs-0 hjGsrv"
    >
      Third value
    </div>
  </div>
</DocumentFragment>
`;
